.travis.yml

```diff
@@ -5,12 +5,15 @@ python:
   - 3.5
   - 3.6
 env:
-  - DJANGO_VERSION=1.8.*
-  - DJANGO_VERSION=1.9.*
   - DJANGO_VERSION=1.10.*
   - DJANGO_VERSION=1.11.*
+  - DJANGO_VERSION=2.0.*
+matrix:
+  exclude:
+  - python: 2.7
+    env: DJANGO_VERSION=2.0.*
 install:
-  - pip install -U pip wheel
+  - pip install -U pip==9.0.* wheel
```

Comment:
https://travis-ci.org/shimizukawa/django-redshift-backend/builds/367612528

Cannot install psycopg2 using python3.6 and pip 10.0.0 from wheel.

Its try to download from pypi and build from source. and its failed

```
2.11s$ pip install -U -r test-requires.txt
Looking in links: /home/travis/build/shimizukawa/django-redshift-backend/./wheelhouse
Obtaining file:///home/travis/build/shimizukawa/django-redshift-backend (from -r test-requires.txt (line 6))
Collecting psycopg2==2.5.5 (from -r test-requires.txt (line 2))
  Downloading https://files.pythonhosted.org/packages/9b/60/f4c79e73a69ded145880bcf4f98eeed741af12c62c5ddc89b754602b1807/psycopg2-2.5.5.tar.gz (779kB)
    100% |████████████████████████████████| 788kB 17.2MB/s 
    Complete output from command python setup.py egg_info:
    running egg_info
    creating pip-egg-info/psycopg2.egg-info
    writing pip-egg-info/psycopg2.egg-info/PKG-INFO
    writing dependency_links to pip-egg-info/psycopg2.egg-info/dependency_links.txt
    writing top-level names to pip-egg-info/psycopg2.egg-info/top_level.txt
    writing manifest file 'pip-egg-info/psycopg2.egg-info/SOURCES.txt'
    Error: could not determine PostgreSQL version from '10.1'
    
    ----------------------------------------
Command "python setup.py egg_info" failed with error code 1 in /tmp/pip-install-ykdfkudz/psycopg2/
```

Reply:
Is master passes build?